templates/feature.html

```diff
@@ -82,44 +82,30 @@
           {% else %}
             Enabled by default
           {% endif %}
-        {% endif %} in <b>desktop Chrome {{ feature.meta.milestone_str }}</b>
+        {% endif %}
+        {% if feature.bug_url %}
+        (<a href="{{ feature.bug_url }}">launch bug</a>)
+        {% endif %}
+        in: </br>
+        {% if feature.shipped_milestone %}
+          - <b>Chrome for desktop release {{ feature.meta.milestone_str }}</b></br>
```

Comment:
But yeah this is consistent with the other UL lists here (also under the documentation section).  But it would be nice (and should be trivial) to add a CSS rule like this to align it better:
```css
#feature ul {
  list-style-position: inside;
}
```